Woodoku/src/ui/command_line_ui.py

```diff
@@ -1,4 +1,4 @@
-from typing import List, Tuple
+from typing import Callable, List, Tuple
 
 from ui.utils import *
```

Comment:
```suggestion
from ui.utils import get_input, red, orange
```
only import what is necessary